### PR TITLE
docs: update Row and Column docs

### PR DESCRIPTION
**Pull request by soumyaloka**
## Description

Removed the style properties from the Column and Row docs and instead referred to the properties in the NavigationManager component as they have the same style properties

## References

LUI-867

### CODE REVIEW

**Review by soumyaloka — COMMENTED** [review]
Thinking to remove few methods from Row and Columns which are already defined in NavigationManager and add a note in docs like
"It inherits all the methods defined in both the NavigationManager and FocusManager components".

File: packages/@lightningjs/ui-components/src/components/Column/Column.mdx [hunk 1]
```diff
@@ -122,11 +122,11 @@ If both of those properties are enabled, `alwaysScroll` will take precedence ove
...
-| name           | type                             | description                            |
-| -------------- | -------------------------------- | -------------------------------------- |
-| itemSpacing    | number                           | spacing between items                  |
-| scrollIndex    | number                           | index of currently selected item       |
-| itemTransition | <DocsLink id="lng.Transition" /> | transition to apply to items on render |
+The Column component shares few same style properties as the one's listed in NavigationManager. For a detailed description, please refer to style properties section in the [NavigationManager component](?path=/docs/navigation-navigationmanager--navigationmanager)
+
+scrollIndex: number
+itemSpacing: number
+itemTransition: <DocsLink id="lng.Transition" />
```
Comment (ImCoolNowRight) [inline packages/@lightningjs/ui-components/src/components/Column/Column.mdx:129]: I think this snuck its way in.

File: packages/@lightningjs/ui-components/src/components/Column/Column.mdx [hunk 2]
```diff
@@ -122,11 +122,11 @@ If both of those properties are enabled, `alwaysScroll` will take precedence ove
 
 ### Style Properties
 
-| name           | type                             | description                            |
-| -------------- | -------------------------------- | -------------------------------------- |
-| itemSpacing    | number                           | spacing between items                  |
-| scrollIndex    | number                           | index of currently selected item       |
-| itemTransition | <DocsLink id="lng.Transition" /> | transition to apply to items on render |
+The Column component shares few same style properties as the one's listed in NavigationManager. For a detailed description, please refer to style properties section in the [NavigationManager component](?path=/docs/navigation-navigationmanager--navigationmanager)
```
Comment (ImCoolNowRight) [inline packages/@lightningjs/ui-components/src/components/Column/Column.mdx]: Can we change this to:

> The Column component shares the same style properties as the ones listed in NavigationManager. For a detailed description, please refer to style properties section in the [NavigationManager component](?path=/docs/navigation-navigationmanager--navigationmanager)

Just a change in the wording and punctuation.

File: packages/@lightningjs/ui-components/src/components/Row/Row.mdx [hunk 3]
```diff
@@ -76,13 +76,7 @@ If multiple of these properties are enabled, the order of precedence from highes
...
-| neverScroll    | boolean                          | when true, the row will never scroll, unless alwaysScroll is set to true, and if false, the row will apply normal scrolling logic', |
-| scrollIndex    | number                           | index of currently selected item                                                                                                    |
-| itemSpacing    | number                           | spacing between items                                                                                                               |
-| itemTransition | <DocsLink id="lng.Transition" /> | transition to apply to items on render                                                                                              |
+The Row component shares the same style properties as the one's listed in NavigationManager.. For a detailed description, please refer to style properties section in the [NavigationManager component](?path=/docs/navigation-navigationmanager--navigationmanager)
```
Comment (ImCoolNowRight) [inline packages/@lightningjs/ui-components/src/components/Row/Row.mdx]: Pasting this here as well so they're in sync.

> The Row component shares the same style properties as the ones listed in NavigationManager. For a detailed description, please refer to style properties section in the [NavigationManager component](?path=/docs/navigation-navigationmanager--navigationmanager)

**Review by ImCoolNowRight — CHANGES_REQUESTED** [review]

**Comment by ImCoolNowRight** [comment]
> Thinking to remove few methods from Row and Columns which are already defined in NavigationManager and add a note in docs like "It inherits all the methods defined in both the NavigationManager and FocusManager components".

I think that totally makes sense and will be easier to maintain when there are updates to be made to only have one place to make them.

**Review by erautenberg — APPROVED** [review]

**Review by ImCoolNowRight — APPROVED** [review]

**Pull request merged**